utils/sql_connecter.py

```diff
@@ -37,32 +46,36 @@ def execute(self, query, type):
         elif type == 'INSERT':
             self.conn.commit()
 
-    def insert_skull_info(self):
-        with open(self.file_path, 'r') as f:
-            reader = csv.DictReader(f)
-            for row in reader:
-                print(row)
-        return None
-
-    def insert_skull_info_test(self, header):
-        df= pd.read_csv(self.file_path, header=header)
-        for index, row in df.iterrows():
-            print(row)
-
-        statement = "INSERT INTO skull VALUES ('{}', '{}', {});".format(row[0], row[1], row[2])
-        print(statement)
-        self.execute(statement, 'INSERT')
-        self.cursor.commit()
-        return None
+    def bulk_insert_csv(self, table_name, header):
```

Comment:
@briyanii pls review this function for your modification in phase1.py (**note that you need to provide the headers of csv (if any) and there are only 3 columns that will be processed (member, time(HH:MM:SS format) and episode no.**)